js.es5.md

```diff
@@ -1,6 +1,6 @@
-# K&L JavaScript Style Guide
+# Education ES5 JavaScript Style Guide
```

Comment:
We we're having a discussion yesterday about the style guide and the most efficient Es6 styles as part of a heated PR review (much wrath wow).
Things like `let` scope and where you put the variable definition came up. Won't break in an  Es5 style, but some prefer it updated for optimal Es6.